irohad/ordering/impl/ordering_service_impl.hpp

```diff
@@ -133,6 +127,12 @@ namespace iroha {
 
       std::unique_ptr<shared_model::interface::ProposalFactory> factory_;
 
+      /**
```

Comment:
Why are these changes required?
I don't force you to revert it, I am just curious.

Reply:
it caused a warning with order of fields initialization